Application/registerationForm.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Form Application</title>
+    <link rel="stylesheet" href="registerationForm.css">
+  </head>
+  <body>
+    <div id ="registerationForm">
+      <h1>Student Form </h1>
+      <p>(Note : Asterisk (*) fields should be filled.)</p>
+      <form id ="myForm" >
+
+       <label for="Name">* Student Name :</label>
```

Comment:
The * symbol should be on the right side as a superscript  (eg: Sudent Name*), usually it will be in red color. Do this change for all the following fields also

Reply:
And one more thing, I'm not able to submit the registration form